addons/l10n_mx/models/account_account.py

```diff
@@ -1,4 +1,1091 @@
+    "604.79",
+    "604.80",
+    "604.81",
+    "604.82",
+    "605",
+    "605.01",
+    "605.02",
+    "605.03",
+    "605.04",
+    "605.05",
+    "605.06",
+    "605.07",
+    "605.08",
+    "605.09",
+    "605.10",
+    "605.11",
+    "605.12",
+    "605.13",
+    "605.14",
+    "605.15",
+    "605.16",
+    "605.17",
+    "605.18",
+    "605.19",
+    "605.20",
+    "605.21",
+    "605.22",
+    "605.23",
+    "605.24",
+    "605.25",
+    "605.26",
+    "605.27",
+    "605.28",
+    "605.29",
+    "605.30",
+    "605.31",
+    "606",
+    "606.01",
+    "607",
+    "607.01",
+    "608",
+    "608.01",
+    "609",
+    "609.01",
+    "610",
+    "610.01",
+    "611",
+    "611.01",
+    "611.02",
+    "612",
+    "612.01",
+    "613",
+    "613.01",
+    "613.02",
+    "613.03",
+    "613.04",
+    "613.05",
+    "613.06",
+    "613.07",
+    "613.08",
+    "613.09",
+    "613.10",
+    "613.11",
+    "613.12",
+    "613.13",
+    "613.14",
+    "613.15",
+    "613.16",
+    "613.17",
+    "613.18",
+    "614",
+    "614.01",
+    "614.02",
+    "614.03",
+    "614.04",
+    "614.05",
+    "614.06",
+    "614.07",
+    "614.08",
+    "614.09",
+    "614.10",
+    "700",
+    "701",
+    "701.01",
+    "701.02",
+    "701.03",
+    "701.04",
+    "701.05",
+    "701.06",
+    "701.07",
+    "701.08",
+    "701.09",
+    "701.10",
+    "701.11",
+    "702",
+    "702.01",
+    "702.02",
+    "702.03",
+    "702.04",
+    "702.05",
+    "702.06",
+    "702.07",
+    "702.08",
+    "702.09",
+    "702.10",
+    "703",
+    "703.01",
+    "703.02",
+    "703.03",
+    "703.04",
+    "703.05",
+    "703.06",
+    "703.07",
+    "703.08",
+    "703.09",
+    "703.10",
+    "703.11",
+    "703.12",
+    "703.13",
+    "703.14",
+    "703.15",
+    "703.16",
+    "703.17",
+    "703.18",
+    "703.19",
+    "703.20",
+    "703.21",
+    "704",
+    "704.01",
+    "704.02",
+    "704.03",
+    "704.04",
+    "704.05",
+    "704.06",
+    "704.07",
+    "704.08",
+    "704.09",
+    "704.10",
+    "704.11",
+    "704.12",
+    "704.13",
+    "704.14",
+    "704.15",
+    "704.16",
+    "704.17",
+    "704.18",
+    "704.19",
+    "704.20",
+    "704.21",
+    "704.22",
+    "704.23",
+    "800",
+    "801",
+    "801.01",
+    "801.02",
+    "802",
+    "802.01",
+    "802.02",
+    "803",
+    "803.01",
+    "803.02",
+    "804",
+    "804.01",
+    "804.02",
+    "805",
+    "805.01",
+    "805.02",
+    "806",
+    "806.01",
+    "806.02",
+    "807",
+    "807.01",
+    "807.02",
+    "808",
+    "808.01",
+    "808.02",
+    "809",
+    "809.01",
+    "809.02",
+    "810",
+    "810.01",
+    "810.02",
+    "811",
+    "811.01",
+    "811.02",
+    "812",
+    "812.01",
+    "812.02",
+    "813",
+    "813.01",
+    "813.02",
+    "814",
+    "814.01",
+    "814.02",
+    "815",
+    "815.01",
+    "815.02",
+    "816",
+    "816.01",
+    "816.02",
+    "899",
+    "899.01",
+    "899.02",
+]
+THIRD_SECTION = re.compile(r'.\d{1,2}[1-9]')
```

Comment:
`.` is a special character in regexp that means any character, this should probably be:

```suggestion
THIRD_SECTION = re.compile(r'\.\d{1,2}[1-9]')
```